src/main/java/org/takes/facets/flash/RsFlash.java

```diff
@@ -100,12 +99,7 @@
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 0.1
- * @todo #686:15min After upgrade to qulice 0.16.4 we have to fix
- *  SingularField PMD warings in this class. Fix this warning
- *  and rid of the PMD suppression here.
  */
-@SuppressWarnings("PMD.SingularField")
-@ToString(callSuper = true, of = "text")
```

Comment:
@mkordas I think that's not necessary once the superclasses don't have non-static fields.